src/tools/clippy/src/driver.rs

```diff
@@ -369,7 +369,6 @@ pub fn main() {
         // Setting RUSTC_WRAPPER causes Cargo to pass 'rustc' as the first argument.
         // We're invoking the compiler programmatically, so we ignore this/
         let wrapper_mode = orig_args.get(1).map(Path::new).and_then(Path::file_stem) == Some("rustc".as_ref());
-
```

Comment:
Please keep this line. @jyn514 was talking about the extra space you added on this line.

Reply:
Sorry, but I tried a few times to remove the whitespace. but couldn't get rid of it.

Reply:
What editor are you using?

Reply:
Neovim, never mind figured it out, removed whitespace.